Makefile

```diff
@@ -1,20 +1,35 @@
 PREFIX ?= /usr/local
-MAN_PREFIX ?= ${PREFIX}/man
+BINDIR ?= $(PREFIX)/bin
+MANDIR ?= $(PREFIX)/man
+SHAREDIR ?= $(PREFIX)/share
+
+.PHONY: all install test distrib
 
 all: target/release/pizauth
 
 target/release/pizauth:
 	cargo build --release
 
+PLATFORM=$(shell uname)
+
 install: target/release/pizauth
-	install -d ${PREFIX}/bin
-	install -c -m 555 target/release/pizauth ${PREFIX}/bin/pizauth
-	install -d ${MAN_PREFIX}/man1
-	install -d ${MAN_PREFIX}/man5
-	install -c -m 444 pizauth.1 ${MAN_PREFIX}/man1/pizauth.1
-	install -c -m 444 pizauth.conf.5 ${MAN_PREFIX}/man5/pizauth.conf.5
-	install -d ${PREFIX}/share/examples/pizauth
-	install -c -m 444 pizauth.conf.example ${PREFIX}/share/examples/pizauth
+	install -d ${DESTDIR}${BINDIR}
+	install -c -m 555 target/release/pizauth ${DESTDIR}${PREFIX}/bin/pizauth
+	install -d ${DESTDIR}${MANDIR}/man1
+	install -d ${DESTDIR}${MANDIR}/man5
+	install -c -m 444 pizauth.1 ${DESTDIR}${MANDIR}/man1/pizauth.1
+	install -c -m 444 pizauth.conf.5 ${DESTDIR}${MANDIR}/man5/pizauth.conf.5
+	install -d ${DESTDIR}${SHAREDIR}/examples/pizauth
```

Comment:
Might be a BSDism, but on my system `pizauth` is the only package installing examples to `/usr/share/examples` -- everyone else uses `/usr/share/$pkgname` (optionally with an `examples` subdirectory`). Can patch this in my packaging if you're attached to this layout.